src/frontend/src/components/importer/ImporterColumnSelector.tsx

```diff
@@ -126,14 +126,79 @@ function ImporterDefaultField({
   );
 }
 
+function ImporterPartImportAsSelect({
```

Comment:
This should be `ImporterImportAsSelect` - it is not specific to importing parts, it should be generic across any type of import